.github/workflows/git_page.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   github-pages:
-    runs-on: ubuntu-latest
```

Comment:
graphviz for ubuntu is buggy, hence the switch to macos env

Reply:
Let’s put that link where you found it's a bug in ubuntu as a comment in the git_page.yml file.

Reply:
Git action error: [link](https://github.com/broadinstitute/long-read-pipelines/actions/runs/4682797742/jobs/8297100575)
Forum posts: [link1](https://gitlab.com/graphviz/graphviz/-/issues/1323) [link2](https://gitlab.com/graphviz/graphviz/-/issues/274)